### ISSUE TITLE
A-level student progress, summer 2017

### ISSUE THREAD

**Issue by PBCranwell** [OpenSourceMalaria/OSM_To_Do_List#541]
During the summer I had 2 A-level students (Ewen and Sam) work with me to continue the work that @odohertymatthew, @TomFreeman3, Alex and Will [started](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/439). 

The first thing that Ewen set out to do was to see why the cyclisation of the hydrazone was unsuccessful in the previous team's hands. To do this, Ewen first had to repeat the previous synthesis of the hydrazone cyclisation precursor. 
![scheme 1](https://user-images.githubusercontent.com/22921225/30847992-6814162e-a295-11e7-977f-0df260073ba2.jpg)

He achieved the SNAr reaction in good yield, which aligned with Matt's [work](http://malaria.ourexperiment.org/reading_university_/15272/Nucleophilic_aromatic_substitution_Synthesis_of_4morpholinobenzaldehyde_N15_Scale_up.html) and the hydrazone formation also went well. It was noted during this reaction that if all of the starting aldehyde was dissolved _before_ adding the hydrazine, the product hydrazone NMR was much better. Cylisation, however, was not successful and the crude NMR data showed a large amount of decomposition and polymerisation products. We suspected that the issue was the presence of the morpholine nitrogen in the benzylic position, therefore we decided to pursue an alternative approach without this. 

The new route is as follows. 
![scheme 2](https://user-images.githubusercontent.com/22921225/30848176-300d8b10-a296-11e7-8d58-9f98f36358a0.jpg)
Condensation of the hydrazone with p-brombenzaldehyde was successful and provided the cyclisation precursor in good yield and high purity. Cyclisation using BAIB in dichloromethane was also successful, albeit in a moderate yield. NMR studies for the cyclisation were undertaken on this substrate (20 mg hydrazone in CDCl3, with 1.1 eq of BAIB) that showed the cyclisation itself is very quick at room temperature (the reaction went to completion within 2.5 hours, and after this time there was no further change to the 1H NMR spectrum), but seemingly only 1 hydrazone isomer cyclises. This will be looked into in more detail in due course, but for now we are happy to be able to bring material through! At this point Ewen's 2 weeks were up, and he left the project. 

Sam was with me for 1 week, and his first task was to prepare more material using Ewen's route, and then look into the SNAr reaction to install the side-chain on the top left. To probe reactivity, the first attempt was undertaken using benzyl alcohol using previously disclosed [conditions](http://malaria.ourexperiment.org/triazolopyrazine_se/9448/Synthesis_of_45phenethoxy124triazolo43apyrazin3ylbenzonitrile_JU_61.html) as it was cheap and readily available. This reaction was quite slow and took a few days to reach completion however, it was successful with the functionalised material prepared in quantitative yield. The final step is to attempt a Buchwald Hartwig coupling to install the requisite amine onto the aromatic ring. This work will be continued by a group of University of Reading BSc students (introductory posts to follow!).
![scheme 3](https://user-images.githubusercontent.com/22921225/30848705-d2091b40-a297-11e7-903c-8df1cb7fcbc8.jpg)

Questions:
1. What alkyl group would be best to install by SNAr to the top-left of the core? At present we tried benzyl alcohol, which works (and is cheap!) but what would be best for the project?
2. Are there any preferences for the amine for the Buchwald-Hartwig coupling?

**Comment by MFernflower** [comment]
If you browse around our s4 compound library you will see that most compounds have a n-phenylethanol fragment in them @pbcranwell

**Comment by PBCranwell** [comment]
@MFernflower yes I saw those - the 2-(3,4-difluorophenyl)ethanol one looks like it could be good but it is rather expensive!! Is there a reliable prep/a cheap supplier available?

**Comment by MFernflower** [comment]
Most cost effective solution is to just couple to 2-phenylethanol @pbcranwell

**Comment by PBCranwell** [comment]
@MFernflower OK, so the fluorines aren't too important then? I can deal with 2-phenylethanol - that is cheap! I will see if anyone else makes any suggestions.

**Comment by MFernflower** [comment]
Just curious: Do you plan on making any more members of this series or just that 1 morpholine? @pbcranwell

**Comment by david1597** [comment]
Hi @PBCranwell - good work!

Q1: Groups with a two-carbon linker in the top-left corner would probably be best. A cheap and readily available way of accessing these is to take the corresponding phenyl acetic acids and reduce them to the alcohol you require. Agreed the 3,4-fluoro substituted phenyl - or similar - could work well. Cheap suppliers include [combi-blocks](http://www.combi-blocks.com/cgi-bin/find.cgi?OR-1998) or [matrix scientific](http://www.matrixscientific.com/005581.html) - about $15 for 5g.

Q2: Less sure here. Maybe just other saturated nitrogen heterocycles?

**Comment by MFernflower** [comment]
@david1597 Great catch on seeing that the acid is cheap!

@PBCranwell Aside from your target morpholine compound - could you buy a few ml of azetidine?

**Comment by alintheopen** [comment]
Lovely work Phil!

**Comment by PBCranwell** [comment]
@david1597 thanks very much. Will get the students onto it!
@MFernflower I was thinking morpholine, pyrrolidine and/or piperidine

**Comment by MFernflower** [comment]
@PBCranwell A few possible compounds to consider making with your students! 
![cranwell_amines](https://user-images.githubusercontent.com/3164942/30936630-801705d4-a3a2-11e7-937e-8c37f6c952d9.png)
